app/screens/custom_status/components/custom_status_input.tsx

```diff
@@ -83,11 +80,11 @@ const CustomStatusInput = ({emoji, isStatusSet, onChangeText, onClearHandle, onO
                     style={style.input}
                     secureTextEntry={false}
                     underlineColorAndroid='transparent'
+                    multiline={true}
```

Comment:
Do we want custom status to be multiline? Or am I missing something?

Reply:
no, just the display of it to wrap around and use multiple lines instead of scrolling

Reply:
I couldn't add line breaks into the status using the iOS simulator.
I'll double check with a real device to make sure

Reply:
even if the iOS simulator does not allow you to add line breaks, the Android one does.

I cannot seem to find a way to wrap the text in the input in multiple lines without having multi-line enabled.

Perhaps what you can do is before submitting the text of the status, you could just modify it to remove all line breaks if there are any and potentially replace them with spaces.

Reply:
it seems external keyboards on android are wired differently,
added a check onTextChange to remove any new lines

Sadly this generates a flicker, and looks like everyone is having the same problem.
![CleanShot 2024-11-11 at 16 04 58](https://github.com/user-attachments/assets/c4b4908e-cd79-4431-8c7a-33c9d5451c73)

Reply:
Not there.. before sendin the text to the action or in the action itself

Reply:
moved to the reducer action